internal/cli/metrics_opt.go

```diff
@@ -14,7 +14,13 @@
 
 package cli
 
-import atlas "go.mongodb.org/atlas/mongodbatlas"
+import (
+	"fmt"
+
+	"github.com/mongodb/mongocli/internal/flag"
+
+	atlas "go.mongodb.org/atlas/mongodbatlas"
```

Comment:
[nit] extra space, gofmt sometimes struggle with the extra spaces 
```suggestion
	"github.com/mongodb/mongocli/internal/flag"
	atlas "go.mongodb.org/atlas/mongodbatlas"
```